src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String USER = "root";
+    private static final String PASSWORD = "1602";
+    private static final String DRIVER = "com.mysql.cj.jdbc.Driver";
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.setProperty("user", USER);
+        DB_PROPERTIES.setProperty("password", PASSWORD);
```

Comment:
Critical point, don't show private information to all people!

Reply:
**critical**: **NEVER** commit and push your real credentials to the remote repository, just remove your creds before committing this file or place some dummy string.